ReactS3Uploader.js

```diff
@@ -32,7 +32,7 @@ var ReactS3Uploader = React.createClass({
     },
 
     uploadFile: function() {
-        new S3Upload({
+        this.S3Upload = new S3Upload({
```

Comment:
Let's name this with a lowercase `s`, since uppercase is typically class names, and this would be an instance of a class.

    this.s3Upload = ...